openedx/core/djangoapps/import_export/courses/tests/utils.py

```diff
@@ -0,0 +1,103 @@
+# pylint: disable=E1101
```

Comment:
Please don't disable warnings like this. Pylint warnings are each identified by a alphanumeric code (C0112) and a symbolic name (empty-docstring). Please use the symbolic names in new code or when updating existing code.

Please scope your suppressions as narrowly as possible. Prefer line-level suppressions the majority of the time.